scalameta/trees/src/main/scala/scala/meta/internal/ast/Helpers.scala

```diff
@@ -146,11 +146,16 @@ object Helpers {
   }
 
   implicit class XtensionMods(mods: List[Mod]) {
-    def has[T <: Mod](implicit tag: ClassTag[T]): Boolean =
-      mods.exists { _.getClass == tag.runtimeClass }
-    def getAll[T <: Mod](implicit tag: ClassTag[T]): List[T] =
-      mods.collect { case m if m.getClass == tag.runtimeClass => m.require[T] }
```

Comment:
Note for the reviewer: As described in the first commit, these methods weren't working as expected.